src/Microsoft.AspNetCore.SystemWebAdapters.FrameworkServices/SessionState/RemoteSession/RemoteAppSessionStateExtensions.cs

```diff
@@ -19,7 +19,6 @@ public static ISystemWebAdapterRemoteServerAppBuilder AddSessionServer(this ISys
 
         builder.Services.AddScoped<IHttpModule, RemoteSessionModule>();
         builder.Services.AddSingleton<ILockedSessionCache, InMemoryLockedSessions>();
-        builder.Services.AddSingleton<ISessionSerializer, BinarySessionSerializer>();
```

Comment:
?

Reply:
Whoops - didn't realize I hadn't reverted that.